### PR TITLE
Go: fix agentURL startup log

### PR DESCRIPTION
**Pull request by rachelyangdog**
## Motivation
Originally, when DD_TRACE_AGENT_URL was set to a valid unix socket (e.g, unix:///var/run/datadog/apm.socket), the startup log on dd-trace-go contains: "agent_url":"http://uds__var_run_datadog_apm.socket/v0.4/traces". We wanted the startup log to reflect the actual, un-normalized socket value instead. This was marked as a bug in system-tests for Go, but has since been fixed in dd-trace-go. 

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/parametric/test_config_consistency.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """
 Test configuration consistency for features across supported APM SDKs.
 """
+
```
Comment (mabdinur) [inline tests/parametric/test_config_consistency.py:4]:
```suggestion
```

**Review by mabdinur — APPROVED** [review]
Can we update the PR description. Aside from that LGTM

File: tests/parametric/test_config_consistency.py [hunk 2]
```diff
@@ -123,12 +124,11 @@ def test_specific_env(self, library_env, test_agent, test_library):
 @features.tracing_configuration_consistency
 class Test_Config_TraceAgentURL:
     """
-    DD_TRACE_AGENT_URL is validated using the tracer configuration. 
+    DD_TRACE_AGENT_URL is validated using the tracer configuration.
     This approach avoids the need to modify the setup file to create additional containers at the specified URL,
     which would be unnecessarily complex.
     """
 
-    @bug(context.library == "golang", reason="APMAPI-390")
```
Comment (mtoffl01) [inline tests/parametric/test_config_consistency.py:131]: Be sure to update APMAPI-390 accordingly.

**Review by mtoffl01 — CHANGES_REQUESTED** [review]
Make sure to update the Golang manifest file (manifests/golang.yml) to specify the version of dd-trace-go where this fix will be included.
(Probably 1.70.0-dev).

**Review by ZStriker19 — APPROVED** [review]

**Review by mtoffl01 — APPROVED** [review]

**Pull request merged**